torch/testing/_internal/common_utils.py

```diff
@@ -753,7 +766,10 @@ def run_tests(argv=UNITTEST_ARGS):
                 [test_case_full_name]
             )
             string_cmd = " ".join(cmd)
-            exitcode = shell(cmd)
+
+            timeout = None if RERUN_DISABLED_TESTS else 15 * 60
```

Comment:
Why not just use the same THRESHOLD x2 here for consistency?

Reply:
I was originally hoping to put a smaller timeout here since --subprocess runs one test at a time, also I would have to add a bunch of imports